ckanext/unaids/theme/templates/package/snippets/bulk_file_uploader.html

```diff
@@ -25,8 +25,8 @@ <h4 class="modal-title">{{ _('Bulk Upload') }}</h4>
                             id="BulkFileUploaderComponent"
                             data-lfsServer="{{ h.extstorage_server_url() }}"
                             data-maxResourceSize="{{ h.max_resource_size() }}"
-                            data-orgId="{{ h.extstorage_resource_storage_prefix(pkg_name).split('/')[0] }}"
-                            data-datasetId="{{ pkg.id }}"
+                            data-orgId="{{ h.get_package_from_id(pkg.id).organization.name }}"
```

Comment:
if you have `pkg` here can't just immediately do `pkg.organization.name` or `pkg.owner_org`?